src/extensions/otel_viewer_extension/Dashboard.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Azure.AI.Details.Common.CLI;
+
+namespace Azure.AI.Details.Common.CLI.Extensions.Otel
+{
+    public class Dashboard
+    {
+        // Execute a command in the command prompt and return the output
+        private static string ExecuteCommand(string command)
+        {
+            ProcessStartInfo processStartInfo = new ProcessStartInfo();
+
+            var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            if (isWindows)
+            {
+                processStartInfo.FileName = "cmd.exe";
+                processStartInfo.Arguments = $"/c {command}";
+            }
+            else 
+            {
+                processStartInfo.FileName = "/bin/bash";
+                processStartInfo.Arguments = $"-c \"{command}\"";
+            }
+           
+            processStartInfo.RedirectStandardOutput = true;
+            processStartInfo.RedirectStandardError = true;
+            processStartInfo.UseShellExecute = false;
+            processStartInfo.CreateNoWindow = true;
+
+            using (Process process = new Process())
+            {
+                process.StartInfo = processStartInfo;
+                process.Start();
+                process.WaitForExit();
+                return process.StandardOutput.ReadToEnd();
+            }
+        }
+        public static void StartDashboard()
+        {
+            // Commands for the terminal
+            string dockerStartCommand = "docker run --rm -it -p 18888:18888 -p 4317:18889 -d --name aspire-dashboard mcr.microsoft.com/dotnet/aspire-dashboard:8.0.0";
+            string dockerLogsCommand = "docker logs aspire-dashboard";
+
+            try
+            {
+                // Start the Docker container
+                string startResult = ExecuteCommand(dockerStartCommand);
+                Console.WriteLine(startResult);
+
+                // Wait before fetching logs if needed
+                System.Threading.Thread.Sleep(3000);
```

Comment:
Is there a reason it's 3s instead of 2s instead of 4s? Is there a more "deterministic" way to determine that the code has waited enough?